testsuite/features/build_validation/add_custom_repositories/add_ubuntu_repositories.feature

```diff
@@ -1,6 +1,7 @@
 # Copyright 2021-2023 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+@ubuntu1804_minion
```

Comment:
Actually you can let only that tag at feature level and get rid of the tags at scenario level.

Reply:
Same remark, we had 16.04 in same file in the old times.